### ISSUE TITLE
Error on compilation

### ISSUE THREAD

**Issue by pksjce** [convoyinc/webpack-js-evaluation-marks#1]
When I install the plugin, add it to my webpack config and then run my compilation, I get the following error.
```
                var name = chunk.name.replace(/\./g, '_');
                                      ^

TypeError: Cannot read property 'replace' of null
    at /Users/pkodmad/Workspace/jira-frontend/node_modules/webpack-js-evaluation-marks/src/index.js:16:39
    at Array.forEach (<anonymous>)
    at Compiler.<anonymous> (/Users/pkodmad/Workspace/jira-frontend/node_modules/webpack-js-evaluation-marks/src/index.js:14:32)
    at Compiler.applyPluginsAsyncSeries (/Users/pkodmad/Workspace/jira-frontend/node_modules/tapable/lib/Tapable.js:206:13)
    at Compiler.emitAssets (/Users/pkodmad/Workspace/jira-frontend/node_modules/webpack/lib/Compiler.js:354:8)
    at onCompiled (/Users/pkodmad/Workspace/jira-frontend/node_modules/webpack/lib/Compiler.js:58:19)
    at applyPluginsAsync.err (/Users/pkodmad/Workspace/jira-frontend/node_modules/webpack/lib/Compiler.js:510:14)
    at next (/Users/pkodmad/Workspace/jira-frontend/node_modules/tapable/lib/Tapable.js:202:11)
    at Compiler.<anonymous> (/Users/pkodmad/Workspace/jira-frontend/node_modules/webpack/lib/CachePlugin.js:78:5)
    at Compiler.applyPluginsAsyncSeries (/Users/pkodmad/Workspace/jira-frontend/node_modules/tapable/lib/Tapable.js:206:13)
```

**Comment by pksjce** [comment]
This seems to happen when a dependency of my project creates an async bundle inside itself.

**Comment by pksjce** [comment]
Fixed by #2
